### PR TITLE
Give the cluster perms to create load balancers.

### PR DESCRIPTION
**Pull request by dandds**
In order for the cluster app registration to create new load balancers,
it needs to have the Network Contributor role for the virtual network.
In the future, we should create a custom policy scoped to exactly the
permissions the cluster needs, per:

https://docs.microsoft.com/en-us/azure/aks/configure-azure-cni#prerequisites

Screen Shot 2020-01-30 at 4 49 19 PM

### CODE REVIEW

File: terraform/modules/k8s/main.tf [hunk 1]
```diff
@@ -81,3 +81,9 @@ resource "azurerm_monitor_diagnostic_setting" "k8s_diagnostic-1" {
     }
   }
 }
+
+resource "azurerm_role_assignment" "k8s_network_contrib" {
+  scope                = var.vnet_id
```
Comment (robgil-dds) [inline terraform/modules/k8s/main.tf]: We already have the vnet id configured in this module. 

https://github.com/dod-ccpo/atst/blob/ca1d26cc672434d7c6ddff3917737fd8e72548a5/terraform/modules/k8s/variables.tf#L32-L35

File: terraform/modules/k8s/variables.tf [hunk 2]
```diff
@@ -66,4 +66,9 @@ variable "client_secret" {
 variable "workspace_id" {
   description = "Log Analytics workspace for this resource to log to"
   type        = string
-}
\ No newline at end of file
+}
+
+variable "vnet_id" {
```
Comment (robgil-dds) [inline terraform/modules/k8s/variables.tf]: See above

File: terraform/modules/k8s/main.tf (hunk 1 above)
Comment (dandds) [reply terraform/modules/k8s/main.tf]: It looks like this is the ID for the private subnet; I need the ID for the whole VNET for the role assignment.

Comment (robgil-dds) [reply terraform/modules/k8s/main.tf]: Yup this makes sense. I missed that. :+1:

**Pull request merged**